source/Superluminal-Model/SetAuthorizationHeaderCommand.class.st

```diff
@@ -0,0 +1,50 @@
+"
+I'm an HTTP Request command. I will set the Authorization header in the HTTP request once applied.
+
+## HTTP Reference
+The HTTP Authorization request header contains the credentials to authenticate a user agent with a server, usually, but not necessarily, after the server has responded with a 401 Unauthorized status and the WWW-Authenticate header.
+Authorization: <type> <credentials>
+Directives
+- <type>
+    Authentication type. A common type is ""Basic"". See [IANA registry of Authentication schemes](http://www.iana.org/assignments/http-authschemes/http-authschemes.xhtml)
+- <credentials>
+    If the ""Basic"" authentication scheme is used, the credentials are constructed like this: The username and the password are combined with a colon (aladdin:opensesame). The resulting string is base64 encoded (YWxhZGRpbjpvcGVuc2VzYW1l).
+
+"
+Class {
+	#name : #SetAuthorizationHeaderCommand,
+	#superclass : #HttpRequestCommand,
+	#instVars : [
+		'authorizationDirectives'
+	],
+	#category : #'Superluminal-Model-Commands'
+}
+
+{ #category : #'instance creation' }
+SetAuthorizationHeaderCommand class >> to: anAuthorizationDirective [
```

Comment:
I can't find a test obtaining the error message.

Reply:
Added in the following commit.